internal/hcs/process.go

```diff
@@ -187,11 +210,12 @@ func (process *Process) Kill(ctx context.Context) (bool, error) {
 		}
 	}
 	events := processHcsResult(ctx, resultJSON)
-	delivered, err := process.processSignalResult(ctx, err)
+	delivered, err := newProcessHandle.processSignalResult(ctx, err)
 	if err != nil {
-		err = makeProcessError(process, operation, err, events)
+		err = makeProcessError(newProcessHandle, operation, err, events)
 	}
 
+	newProcessHandle.killSignalDelivered = delivered
```

Comment:
We shouldn't really need this, since we close and throw away `newProcessHandle` after this function call, right?

Reply:
Yes, I put it in for readability to indicate it was successfully since we were setting it on the original compute system handle too. Will get rid of it